pennylane/init.py

```diff
@@ -20,6 +20,66 @@
 from pennylane import numpy as np
 
 
+def particle_conserving_u1_uniform(n_layers, n_wires, low=0, high=2 * pi, seed=None):
+    r"""Creates a parameter array for :func:`~.ParticleConservingU1`, drawn from a uniform
+    distribution.
+
+    Each parameter is drawn uniformly at random from between ``low`` and ``high``.
+    The parameters define the trainable angles entering the particle-conserving
+    exchange gates :math:`U_{1,\mathrm{ex}}(\phi, \theta)` implemented by the
+    :func:`~.u1_ex_gate()`.
+
+    Args:
+        n_layers (int): number of layers
+        n_wires (int): number of qubits
+        low (float): minimum value of uniform distribution
+        high (float): maximum value of uniform distribution
+        seed (int): seed used in sampling the parameters, makes function call deterministic
+
+    Returns:
+        array: parameter array
+    """
+
+    if seed is not None:
+        np.random.seed(seed)
+
+    if n_wires < 2:
+        raise ValueError("The number of qubits must be  >= 2; got 'n_wires' = {}".format(n_wires))
+
+    params = np.random.uniform(low=low, high=high, size=(n_layers, n_wires - 1, 2))
+    return params
+
+
+def particle_conserving_u1_normal(n_layers, n_wires, mean=0, std=0.1, seed=None):
+    r"""Creates a parameter array for :func:`~.ParticleConservingU1`, drawn from a normal
+    distribution.
+
+    Each parameter is drawn from a normal distribution with ``mean`` and ``variance``.
```

Comment:
Would variance not have to be std? Or did you take it from the other init functions?